tests/main/auto-refresh-pre-download/task.yaml

```diff
@@ -0,0 +1,109 @@
+summary: Check that an inhibited auto-refresh triggers a pre-download change and resumes on close
+
+# Ubuntu 14.04's special version of systemd doesn't have StartTransientUnit API.
+systems: [-ubuntu-14.04-*]
+
+kill-timeout: 5m
+
+environment:
+  REFRESH_TYPE/close: "close"
+  REFRESH_TYPE/ignore: "ignore-running"
+
+prepare: |
+  snap install --devmode jq
+  snap install test-snapd-sh
+  snap refresh --unhold
```

Comment:
why this? also do we need to resetup the hold in restore?

Reply:
I'm not sure why but the test starts with a hold placed. The only test that places holds is `snap-refresh-hold` and it removes them during the restore stage. If I remove this line, I get this at the beginning of the test even if I run it individually:
```
google:ubuntu-22.04-64 .../tests/main/auto-refresh-pre-download# snap refresh --time
schedule: 00:00-23:59
last: today at 09:56 UTC
hold: today at 11:55 UTC
next: tomorrow at 13:13 UTC
```

Maybe @sergiocazzolato knows why

Reply:
we want to avoid random refreshes as much as possible of things like kernels etc or bases